src/main.dart

```diff
@@ -1,8 +1,157 @@
-// TODO: Run task list app
+import 'dart:io';
+
 void main(List<String> args) {
-  print('Hello World');
+  List<TaskListApp> tasks = [];
+  print("♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡");
+  print('Task List App');
+
+  var answer;
+  String? answer1;
+
+  do {
+    print("♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡");
+    print('What would you like to do?');
+    print('1. Add task');
+    print('2. Edit task');
+    print('3. Delete task');
+    print('4. View tasks');
+    print('5. Exit');
+    int? answer;
+
+    try {
+    answer = int.parse(stdin.readLineSync()!);
+    } catch (e) {
+    print('Invalid input. Please enter a number.');
+    }
+    print("♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡");
+
+    switch (answer) {
+      case 1:
+        do {
+          print('What would you like your task name to be?');
+          String? taskname = stdin.readLineSync();
+
+          print('Enter your task description:');
+          String? description = stdin.readLineSync();
+
+          print('Is you task completed:');
+          bool isCompleted = stdin.readLineSync()?.toLowerCase() == 'y' ||
+              stdin.readLineSync()?.toLowerCase() == 'yes';
+          print("♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡♡");
+
+          TaskListApp task = TaskListApp.withDetails(
+            id: tasks.length + 1,
+            taskname: taskname!,
+            description: description!,
+            isCompleted: isCompleted,
```

Comment:
Download Spell Checker

Reply:
thank you so much for your comments they helped me see all my errors i have fixed all of them(i think) but this one i couldn't figure it out can you please is it something wrong with my spelling or is there something i should implement something here.

Reply:
It's just spelling issues and variable naming issues. There is an extension in vscode called spellchecker, helps you avoid those

Reply:
ohh okay don't worry got it downloaded. i do struggle a bit with spelling in english lol. i will hopefully work on improving my english. thank you!!